plugins/@grouparoo/salesforce/src/lib/export/salesforceExporter.ts

```diff
@@ -338,6 +348,40 @@ function buildUserPayload(
   return { row, referenceData };
 }
 
+async function getForeignKeyType(
+  client: any,
+  cacheData: SalesforceCacheData,
+  recordObject: string,
+  recordMatchField: string
+) {
+  const recordInfo = await describeObject(
+    client,
+    cacheData,
+    recordObject,
+    true
```

Comment:
I think you want `false` here so it actually does the caching.
`passthru?: boolean; // send true to not read or write or lock`

Reply:
It does `true` in the other spot because the mapping page should always be up to date. But this will get called a lot, so it should use the cache.